pyodide-build/pyodide_build/cli/build_oot.py

```diff
@@ -10,13 +10,17 @@ def main(
         "requested",
         help="Which symbols should be exported when linking .so files?",
     ),
+    package: str = typer.Option(
+        "",
+        help="pypi package-spec to download and build",
+    ),
     ctx: typer.Context = typer.Context,
 ) -> None:
     """Use pypa/build to build a Python package"""
     initialize_pyodide_root()
     common.check_emscripten_version()
     backend_flags = ctx.args
-    build.run(exports, backend_flags)
+    build.run(exports, package, backend_flags)
```

Comment:
I think we should have a different subcommand for this. I would like to keep `pyodide build` as a subset of [pypa/build](https://pypa-build.readthedocs.io/en/latest/#python--m-build)'s API. In `pypa/build` the positional argument is a path to the local directory we want to build which I think is more natural. 

Also in favor of using a different time, this current command will just download a wheel from pypi if there is a pure Python wheel, so in that case nothing is built at all and the name is a misnomer.